crates/wasm-mutate/src/mutators/remove_export.rs

```diff
@@ -18,7 +18,12 @@ impl Mutator for RemoveExportMutator {
     ) -> Result<Box<dyn Iterator<Item = Result<Module>> + 'a>> {
         let mut exports = ExportSection::new();
         let mut reader = ExportSectionReader::new(config.info().get_exports_section().data, 0)?;
+
         let max_exports = reader.get_count() as u64;
+        if max_exports == 0 {
+            return Err(Error::no_mutations_applicable());
+        }
```

Comment:
I thought that this was guarded by `can_mutate` below? Otherwise how could `mutate` get called?

Reply:
I didn't check super carefully, mostly just trying to cover all `gen_range` calls.